gtk/src/builder.rs

```diff
@@ -22,7 +22,7 @@ impl Builder {
 
 pub trait BuilderExtManual: 'static {
     #[doc(alias = "gtk_builder_get_object")]
-    fn get_object<T: IsA<Object>>(&self, name: &str) -> Option<T>;
+    fn object<T: IsA<Object>>(&self, name: &str) -> Option<T>;
```

Comment:
Do we want to rename this one? not very sure

Reply:
I don't know.

@GuillaumeGomez @sdroege ?

Reply:
I don't know. Is it common for getters-by-name to omit the `get_` prefix?

Reply:
> Is it common for getters-by-name to omit the get_ prefix?

I guess, that depends, that's the reason why I stuck by the "conservative" approach initially. Here is another example:

https://github.com/fengalin/gtk-rs/blob/f5b0dd01859e9b43139b4329cd915adb20b59a81/gio/src/auto/action_group.rs#L49

Reply:
We can also consider usage: 

```
let header_bar = builder.get_object("header-bar").unwrap();
```

or

```
let header_bar = builder.object("header-bar").unwrap();
```

Reply:
Do we have something similar in some other well-written Rust library or even better in std that we could take as reference?

Reply:
Searching for `get` in `std` doesn't raise a lot: https://doc.rust-lang.org/std/?search=get.

The closet to the name thing would be the [`HashMap::get`](https://doc.rust-lang.org/std/collections/struct.HashMap.html#method.get).

Reply:
I found this from an often praised Rust developer: https://docs.rs/grep-matcher/0.1.4/grep_matcher/trait.Matcher.html#method.capture_index

As a counter example, in this `struct`, `get` is used in some places and note in others: https://docs.rs/wgpu/0.7.1/wgpu/struct.Adapter.html

I found a similar situation in Druid. In a single `struct`, you can find functions retrieving something [with a `get` prefix](https://docs.rs/druid/0.7.0/druid/struct.Clipboard.html#method.get_format) and others [without `get`](https://docs.rs/druid/0.7.0/druid/struct.Clipboard.html#method.preferred_format).

I also searched in `iced`, but its model doesn't favour getters in the first place.

I believe this is really subjective.

Reply:
Continuing my quest: `bevy` seems to be using [`get` when provided with additional arguments](https://docs.rs/bevy/0.5.0/bevy/asset/struct.SourceInfo.html).

Reply:
Hmm, it's really difficult to take a decision here, but I guess let's remove the get prefix here anyway :) at worst, if someone complains, we can re-consider the rename here.